### ISSUE TITLE
Runkeeper export on Linux

### ISSUE THREAD

**Issue by cyberjunky** [altera2015/ttwatcher#6]
Should the runkeeper export option be fully working?
After authorizing and enabling runkeeper using ttwatcher settings with my account (which auth is successful)
ttwatcher hangs until I force to close it, updated settings are not saved.
Have to investigate at which location in the source code this happens.

**Comment by altera2015** [comment]
Can you describe your steps to reproduce? When I do it here auth & export work.

**Comment by altera2015** [comment]
Ah and if you compile your own do you have a runkeeper auth key?

**Comment by cyberjunky** [comment]
Yes I added my auth key to runkeeper_auth.h before compile.

It seems to hang here, like it's having a problem with config write for export tag, but not sure yet.
How does this section look like, maybe I can create my own manually to see if that's the case.

```
void IExporterConfig::writeExportTag(QDomDocument &document, QDomElement &element, const QString &idAttribute)
{
    QDomNodeList nodes = element.elementsByTagName("export");
    for (int i=0;i<nodes.length();i++)
```

**Comment by altera2015** [comment]
Try to catch it under the debugger, just reading that code I don't see how it could hang.

**Comment by cyberjunky** [comment]
I just did a new pull, added my runkeeper id and secret, recompiled.
At first start it automatically tried to export the run to all export, strava, tcx etc even though is didn't enable any of them.
Then I enabled and registered runkeeper, it went ok!
When I clicked on export, I got "export failed" (exact msg not known), after a restart of ttwatcher, it exported all selected run fine! No idea why it works now.
Some note: would be nice to be able to multiselect runs to export, would be nice to be able to see which runs were exported already. (other color, or icon in front/behind it)
Anyway, my runs are exported, and i'm happy.
